models/PyCryptoBot.py

```diff
@@ -481,8 +481,7 @@ def getTakerFee(self):
             return api.getTakerFee()
         elif self.exchange == 'binance':
             api = BAuthAPI(self.getAPIKey(), self.getAPISecret(), self.getAPIURL())
-            #return api.getTakerFee()
-            return 0.005
+            return api.getTakerFee(self.getMarket())
```

Comment:
Calling api.getTakerFee() function with self.getMarket() parameter caused this error:
Sometimes got the error right away after running the bot, some times a little bit later and the other times did not get any error at all. All these happened in --sim mode.

![image](https://user-images.githubusercontent.com/3215691/118411711-9bce2b00-b69e-11eb-9530-6ce781e33847.png)

Reply:
That seems to be unrelated... The bot only calls the API if a market is passed, otherwise, it returns the default fee, and when it makes the API call, the Binance API returns this 'success' message when the request is successful.